pages/full.js

```diff
@@ -8,8 +8,8 @@ import ExploreSvg from 'public/explore.svg'
 import styles from 'styles/full.module.css'
 
```

Comment:
Have you thought about destructuring the styles to prevent duplication? ie.

```
const {
  about,
  contentRoot,
  dark,
  grey,
  hero,
  ...
} = styles

// later in the code
...
<section className{`${hero} {dark}`}>
...
```

You could even destructure the h2 if you like:

```
const {h2} = Heading.VARIANT

Reply:
I hadn't thought about this. There's some other cleanup / organization type stuff I'd like to do once all the content and styles are in (easier to see what's duplicated) and I think this type of thing would be good to do then. I'll definitely keep this suggestion in mind, I think it could be nice :+1: